api/docs/http_certificate_status.md

```diff
@@ -13,7 +13,7 @@ Find
 ----
 
     GET /puppet-ca/v1/certificate_status/:certname?environment=:environment
-    Accept: pson
+    Accept: text/pson
```

Comment:
I couldn't configure puppetserver to allow these requests, so I'm not sure if this is correct? Or if it's `application/json`?